nx/source/runtime/devices/romfs_dev.c

```diff
@@ -17,6 +17,9 @@
 
 typedef struct romfs_mount
 {
+    bool               setup;
+    s32                id;
+    devoptab_t device;
```

Comment:
Spaces for alignment in this struct. Also, probably for alignment reasons `romfs_mount::device` should be the first member, the bools should be together, etc.